src/js/tracks/video-track-enums.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file video-track-enums.js
+ */
+
+/**
+ * https://html.spec.whatwg.org/multipage/embedded-content.html#dom-videotrack-kind
+ *
+ * enum VideoTrackKind {
+ *   "alternative",
+ *   "captions",
+ *   "main",
+ *   "sign",
+ *   "subtitles",
+ *   "commentary",
+ *   "",
+ * };
+ */
+const VideoTrackKind = {
```

Comment:
It's a common idiom to define enums for these sorts of constants in many programming languages but I think it adds unnecessary abstraction here. The arguments in favor tend to come in two primary flavors:

1. It's easier to change with automated tools
2. It's good documentation

Point 1 doesn't hold for javascript because of its dynamic nature-- to be safe you have to do a find-replace anyways. Secondly, these constants are actually strings without a first class enum in the spec. We won't be changing them (beyond addition, maybe) anytime soon.

Point 2 makes a lot of sense but there are other ways of achieving it that don't impact runtime behavior. Our documentation allows you to add stub entries wherever you want that get placed into the generated API docs. That seems a better way of surfacing this info if it's important to users.